readme.md

```diff
@@ -190,6 +190,7 @@ Please read the [contribution guidelines](contributing.md) or the [creating a li
 
 - [Papers We Love](https://github.com/papers-we-love/papers-we-love)
 - [Talks](https://github.com/JanVanRyswyck/awesome-talks)
+- [Mathematics](https://github.com/rossant/awesome-math)
```

Comment:
I initially wrote "Math" then I read "check your spelling" in the contributing guidelines... ;) Just curious, which of the rules is not respected by typing "Mathematics"?